pkg/kubelet/pod_workers.go

```diff
@@ -263,6 +267,9 @@ func (p *podWorkers) wrapUp(uid types.UID, syncErr error) {
 	case syncErr == nil:
 		// No error; requeue at the regular resync interval.
 		p.workQueue.Enqueue(uid, wait.Jitter(p.resyncInterval, workerResyncIntervalJitterFactor))
+	case strings.Contains(syncErr.Error(), "network is not ready"):
```

Comment:
Declare `network is not ready` a constant and use it in both places to avoid unnecessary breakage in the future.

Reply:
done